crates/sui/src/unit_tests/keytool_tests.rs

```diff
@@ -38,7 +38,7 @@ const TEST_MNEMONIC: &str = "result crisp session latin must fruit genuine quest
 #[test]
 fn test_addresses_command() -> Result<(), anyhow::Error> {
     // Add 3 Ed25519 KeyPairs as default
-    let mut keystore = Keystore::from(InMemKeystore::new(3));
+    let mut keystore = Keystore::from(InMemKeystore::new_insecure(3));
```

Comment:
error[E0599]: no function or associated item named `new_insecure` found for struct `InMemKeystore` in the current scope ???